Makefile

```diff
@@ -126,7 +126,9 @@ _tmpdir-rm: ##@prepare Remove TMPDIR
 
 _install-hooks: SHELL := /bin/sh
 _install-hooks: ##@prepare Create prepare-commit-msg git hook symlink
+ifeq (,$(wildcard .git/hooks/prepare-commit-msg))
 	@ln -s ../../scripts/hooks/prepare-commit-msg .git/hooks
```

Comment:
You can achieve the same thing with less code and less file operation by just adding `-f`.